lib/broadway/topology.ex

```diff
@@ -151,12 +150,24 @@ defmodule Broadway.Topology do
       processors_config: init_processors_config(opts[:processors]),
       batchers_config: opts[:batchers],
       context: opts[:context],
-      terminator: :"#{name_prefix(opts[:name])}.Terminator",
       max_restarts: opts[:max_restarts],
       max_seconds: opts[:max_seconds],
       shutdown: opts[:shutdown],
       resubscribe_interval: opts[:resubscribe_interval]
     }
+    |> put_terminator()
+    |> put_rate_limiter(opts)
```

Comment:
Moved `rate_limiter` (its name) into config. This lines it up with eg Terminator and ensures we don't have RateLimiter name itself (as you'll see below)

Lmk if this works

Reply:
This works but have both `terminator: nil` and `rate_limiter: nil` in the initial map and simply update the keys instead. :)

Reply:
Good call